uiServices/src/services_test.go

```diff
@@ -91,6 +134,52 @@ func TestValidJsonPost(t *testing.T) {
 		t.Errorf("TestValidJsonPost. Expected:", http.StatusCreated, " Got:", w.Code, "  Error. Did not succesfully post")
 	}
 }
+
+func TestPostWithInvalidHeader (t *testing.T) {
+	r := chi.NewRouter()
+	r.Mount("/", GetIndexPage())
+
+	reader := strings.NewReader(validJson)
+	r.HandleFunc("/configs", postConfigs)
+
+	filePath := "xxxxxx"
+	request, err := http.NewRequest(http.MethodPost, "/configs", reader)
+	request.Header.Set("configPathDir", filePath)
+
+	w := httptest.NewRecorder()
+	r.ServeHTTP(w, request)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	if w.Code != http.StatusBadRequest {
+		t.Errorf("TestValidJsonPost. Expected:", http.StatusBadRequest ," Got:", w.Code, "  Error. Did not succesfully post")
+	}
+}
+
+func TestInternalServerErrorwithPost (t *testing.T) {
+	r := chi.NewRouter()
+	r.Mount("/", GetIndexPage())
+
+	reader := strings.NewReader(validJson)
+	r.HandleFunc("/configs", postConfigs)
+
+	filePath := "``¬¬``|"
+	request, err := http.NewRequest(http.MethodPost, "/configs", reader)
+	request.Header.Set("configPathDir", filePath)
+
+	w := httptest.NewRecorder()
+	r.ServeHTTP(w, request)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	if w.Code != http.StatusInternalServerError {
+		t.Errorf("TestValidJsonPost. Expected:", http.StatusInternalServerError ," Got:", w.Code, "  Error. Did not succesfully post")
```

Comment:
This test is failing for me.

services_test.go:180: TestValidJsonPost. Expected:%!(EXTRA int=500, string= Got:, int=400, string=  Error. Did not succesfully post)

I also suggest we switch to use https://github.com/stretchr/testify for assertions in unit tests. it's much cleaner.